asm/bss_8001C4D0.s

```diff
@@ -667,76 +667,25 @@ glabel gItemWindowObjectByPlayerId
 glabel D_80183E98
 .skip 8
 
-glabel D_80183EA0
-.skip 4
-
-glabel D_80183EA4
-.skip 4
-
-glabel D_80183EA8
-.skip 4
-
-glabel D_80183EAC
-.skip 4
-
-glabel D_80183EB0
-.skip 16
-
-glabel D_80183EC0
-.skip 8
-
-glabel D_80183EC8
-.skip 20
-
-glabel D_80183EDC
-.skip 16
-
-glabel D_80183EEC
-.skip 60
-
-glabel D_80183F28
-.skip 4
-
-glabel D_80183F2C
-.skip 8
+glabel indexObjectList1
```

Comment:
objectListIndex1

(I think I prefer)
objectListIndex
objectListIndex2
etc.
but not against objectListIndex1

Reply:
by naming it objectListIndex1 this implictly mean there is atleast a 2

Reply:
fair